enterprise/admin/__init__.py

```diff
@@ -48,17 +51,58 @@ class EnterpriseCustomerIdentityProviderInline(admin.StackedInline):
     form = EnterpriseCustomerIdentityProviderAdminForm
 
 
+class EnterpriseCustomerEntitlementInline(admin.StackedInline):
+    """
+    Django admin model for EnterpriseCustomerEntitlement.
+    The admin interface has the ability to edit models on the same page as a parent model. These are called inlines.
+    https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.StackedInline
+    """
+
+    model = EnterpriseCustomerEntitlement
+    extra = 0
+    can_delete = True
+    fields = ('enterprise_customer', 'entitlement_id', 'ecommerce_coupon_url',)
+
+    def ecommerce_coupon_url(self, instance):
+        """
+        Instance is EnterpriseCustomer. Return e-commerce coupon urls.
+        """
+        if not instance.entitlement_id:
+            return "N/A"
```

Comment:
For new data.
<img width="1275" alt="screen shot 2017-01-20 at 6 06 38 pm" src="https://cloud.githubusercontent.com/assets/7334669/22150533/4828c5ee-df3b-11e6-9bfa-246d7a049d31.png">